packages/bpk-component-image/src/BpkImage.js

```diff
@@ -206,6 +218,7 @@ BpkImage.propTypes = {
 };
 
 BpkImage.defaultProps = {
+  borderRadiusStyle: null,
```

Comment:
I think it makes more sense to have a `BORDER_RADIUS_STYLES.none` option which is the default, instead of `null`. It feels like a cleaner API to me if the border radius is always set to one of `BORDER_RADIUS_STYLES`.

Reply:
Done :)